packages/client/webapp/src/components/ProjectCard.vue

```diff
@@ -0,0 +1,137 @@
+<script setup lang="ts">
+// import Button from '@/components/Button.vue';
+import IconList24 from '@carbon/icons-vue/es/list/24';
+import IconAddFilled32 from '@carbon/icons-vue/es/add--filled/32';
+
+interface Props {
+	category: String;
+	projectName: String;
+	isNewProjectCard: Boolean;
+	p: Number;
+}
+
+// Not sure how to get rid of these red underlines - I am following these rules: https://vuejs.org/api/sfc-script-setup.html#typescript-only-features
+const props = withDefaults(defineProps<Props>(), {
+	category: '',
+	projectName: '',
+	isNewProjectCard: false,
+	p: 0
+});
+
+function createNewProject() {}
+</script>
+
+<template>
+	<div v-if="!props.isNewProjectCard" class="project-card">
```

Comment:
In the design @pascaleproulx is using so basic images, but for now, you can design a placeholder icon in case of a missing images. You can use the `No image` icon for that purpose.